__tests__/get-api-categories.test.js

```diff
@@ -47,7 +47,39 @@ describe('Testing the /api/categories', () => {
 
     })
 })
-
+describe('test get/categories ', () => {
+    test('test that get/categories returns the correct outcome', () => {
+        return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((result) => {
+            const {categories} = result.body
+            expect(categories).toBeInstanceOf(Array)
+            const output = {
+                "categories":[
+                {
+                  slug: 'euro game',
+                  description: 'Abstact games that involve little luck'
+                },
+                {
+                  slug: 'social deduction',
+                  description: "Players attempt to uncover each other's hidden role"
+                },
+                {
+                  slug: 'dexterity',
+                  description: 'Games involving physical skill'
+                },
+                {
+                  slug: "children's games",
+                  description: 'Games suitable for children'
+                }
+              ]
+            }
+              
+              expect(result.body).toEqual(output);
+        })
```

Comment:
This is a bit of a brittle test as the categories will be designed to be dynamically changing based on articles added and removed. How might you test that each object has the desired properties regardless of the length of the returned array?